test/framework/functional/output_http.go

```diff
@@ -0,0 +1,99 @@
+package functional
+
+import (
+	"strings"
+
+	log "github.com/ViaQ/logerr/v2/log/static"
+	logging "github.com/openshift/cluster-logging-operator/apis/logging/v1"
+	"github.com/openshift/cluster-logging-operator/internal/constants"
+	"github.com/openshift/cluster-logging-operator/internal/runtime"
+	"github.com/openshift/cluster-logging-operator/internal/utils"
+)
+
+const (
+	//VectorImage          = "quay.io/domino/vector:0.26.X-distroless-static"
+	VectorImage          = "docker.io/timberio/vector:0.25.0-alpine"
+	VectorHttpSourceConf = `
+[sources.my_source]
+type = "http"
+address = "127.0.0.1:8090"
+encoding = "json"
+
+[sinks.my_sink]
+inputs = ["my_source"]
+type = "file"
+path = "/shared/output.logs"
+
+[sinks.my_sink.encoding]
+codec = "json"
+`
+
+	FluentdImage          = "quay.io/openshift-logging/fluentd"
```

Comment:
Can we pull this from know constant we already maintain?

Reply:
removed this variable